x-pack/elastic-agent/_meta/config/common.p2.yml.tmpl

```diff
@@ -41,7 +41,7 @@ inputs:
 #       # When using IP addresses, it is recommended to only use localhost.
 #       host: localhost
 #       # Port on which the HTTP endpoint will bind. Default is 0 meaning feature is disabled.
-#       port: 0
+#       port: 6791
```

Comment:
What is the default port you picked here? Mainly curious, is there some discussions around it?

Reply:
default port was in the PR before. we use 6790 already so i increased +1